src/message-fragmentation.md

```diff
@@ -43,8 +49,12 @@ Note that:
 - Nodes should enforce a limit on the number of fragments per message (see the [parameters
   chapter](./parameters.md)), by simply discarding fragments with a too-large
   `num_fragments_minus_1`.
+// TODO should common parameter like this be part of the runtime api?
```

Comment:
Yes, I do think we will ultimately want parameters like this to come from the runtime. It's on my todo list. For now though I've avoided doing this to keep things simple.